### ISSUE TITLE
Hide Canada from the US and UK sites, or show a warning on the Canada site

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-app#159]
Given it's very early

**Comment by dotslashbit** [comment]
can you provide some clarification:
1. Have you decided where to put the banner and how the banner should look like

**Comment by anth-volk** [comment]
I don't believe there's been any decision. If you'd like, perhaps you could mock something up? That would help us and give you a greater impact upon not just the functioning of PolicyEngine, but also the design and user experience.

**Comment by MaxGhenis** [comment]
Thank you for filing this issue. Closing as part of the `policyengine-app` (v1) wind-down: policyengine.org is now served by [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2), and this repository is being archived, so this issue is frozen against a codebase that no longer deploys anywhere.

If this is still relevant on the current site, please re-file it against [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2) — we'd welcome it there. Apologies that this sat open through the migration.
